core-blocks/latest-comments/style.scss

```diff
@@ -0,0 +1,59 @@
+.wp-block-latest-comments {
+	padding-left: 2.5em;
+
+	&__comment-timestamp {
```

Comment:
`wp-block-latest-comments __comment-timestamp` should be moved to top level as its own definition. It should be also consolidated with rules from lines 23-26.